gammapy/makers/utils.py

```diff
@@ -28,6 +28,74 @@
 log = logging.getLogger(__name__)
 
 
+def _get_fov_coords(pointing, irf, geom, use_region_center=True, obstime=None):
```

Comment:
Is it the right place for this function? It could also live in pointing or irf.

Reply:
@AtreyeeS: should we then follow these recommendations ?

Reply:
For now, I would suggest adding a TODO and open an issue about this.

Reply:
Created #5041